### ISSUE TITLE
Node only version?

### ISSUE THREAD

**Issue by astanciu** [kciter/qart.js#2]
Would be great to have this work in Node. There is a 'canvas' module that simulates the browser canvas

**Comment by evaryont** [comment]
Yeah! It would be awesome to support offline/server-side rendering.

**Comment by kciter** [comment]
Good! I added `TODO` list. :)

**Comment by ccoenen** [comment]
canvas is a slight hassle (as are all `node-gyp` dependent packages). Keep in mind: once you add `canvas` [these become essentially _your_ setup instructions](https://github.com/Automattic/node-canvas/wiki/Installation---Windows).

**Comment by syzer** [comment]
you got node `qrcode` for that, + its cli module
